tests/integration/profiles_management/helpers/profiles.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from pathlib import Path
+from typing import List
+
+import pytest
+from lightkube import Client, codecs
+from lightkube.generic_resource import (
+    GenericGlobalResource,
+    GenericNamespacedResource,
+    create_global_resource,
+)
+
+from tests.integration.profiles_management.helpers import k8s
+
+log = logging.getLogger(__name__)
+
+Profile = create_global_resource(
+    group="kubeflow.org", version="v1", kind="Profile", plural="profiles"
+)
+
+
+def load_profile_from_file(file_path: str, context: dict = {}) -> codecs.AnyResource:
```

Comment:
note: To me it is not obvious what the `codecs.AnyResource` type is, and I couldn't find it in the documentation. I searched `lightkube`'s source code to find it, and apparently it is `Union[GenericGlobalResource, GenericNamespacedResource]`. Maybe we can explain it in an appropriate place.